packages/@ember/-internals/glimmer/lib/syntax/outlet.ts

```diff
@@ -124,6 +125,13 @@ function stateFor(
   if (render === undefined) return null;
   let template = render.template;
   if (template === undefined) return null;
+
+  // this guard can be removed once @ember/test-helpers@1.6.0 has "aged out"
+  // and is no longer considered supported
```

Comment:
So is the intention to still merge and release https://github.com/emberjs/ember-test-helpers/pull/677 or to rather make a new backwards-incompatible release of `@ember/test-helpers` that drops support for all Ember versions prior to https://github.com/emberjs/ember.js/pull/18096? 🤔

Reply:
Ya, I still think we should still land https://github.com/emberjs/ember-test-helpers/pull/677.